client/extensions/woocommerce/app/shipping/shipping-methods/free-shipping.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormCurrencyInput from 'components/forms/form-currency-input';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+
+class FreeShippingMethod extends Component {
+	constructor( props ) {
+		super( props );
+
+		//TODO: use redux state with real data
+		this.state = { everyone: props.everyone };
+
+		this.onOptionChange = this.onOptionChange.bind( this );
+	}
+
+	onOptionChange( event ) {
+		this.setState( { everyone: 'everyone' === event.currentTarget.value } );
+	}
+
+	renderMinSpendBox() {
+		return (
+			<FormCurrencyInput currencySymbolPrefix="$" value={ this.state.minSpend } />
```

Comment:
Should `currencySymbolPrefix` come from state/props as well? _(It will ultimately come from retrieved store settings)_